v2_1/ws/rest/src/sdmx-rest.yaml

```
@@ -222,6 +222,21 @@ paths:
           $ref: '#/components/responses/503'
           
 components:
+  schemas:
+    sdmxPeriod:
```

Style note: I'd use CamelCase for schemas.

```suggestion
    SdmxPeriod:
```